azure/basic-aks-cluster/blueprint.yaml

```diff
@@ -110,9 +110,11 @@ spec:
     type: Input
     prompt: What do you want to name the cluster?
 
-  # Hidden prefix
   - name: Prefix
-    value: azure-aks-
+    type: Input
```

Comment:
does this needs to be prompted, can't it be hidden?

Reply:
Removing Prefix